drivers_windows.go

```diff
@@ -1,9 +1,16 @@
 package libnetwork
 
-import "github.com/docker/libnetwork/drivers/windows"
+import (
+	"github.com/docker/libnetwork/drivers/null"
+	"github.com/docker/libnetwork/drivers/windows"
+)
 
 func getInitializers() []initializer {
 	return []initializer{
-		{windows.Init, "windows"},
+		{null.Init, "null"},
+		{windows.GetInit("Transparent"), "Transparent"},
```

Comment:
Currently the initialization of the IPAM drivers is being explicitly invoked by libnetwork in https://github.com/docker/libnetwork/blob/master/controller.go#L193
and it is platform independent.

This Windows specific code should not take care of the null ipam initialization.